app/src/main/java/net/squanchy/support/lang/Optionals.kt

```diff
@@ -0,0 +1,3 @@
+package net.squanchy.support.lang
+
+fun <T> T?.optional(): Optional<T> = Optional.fromNullable(this)
```

Comment:
We might also add 
`fun <T> Optional<T>.unwrap(): T? = if(isPresent()) get() else null`

Reply:
If we need it, doesn't really seem so far we do